webextensions/api/scripting.json

```diff
@@ -308,8 +307,7 @@
                 },
                 "edge": "mirror",
                 "firefox": {
-                  "version_added": "102",
-                  "notes": "Only supports <code>ISOLATED</code>, not <code>MAIN</code>."
+                  "version_added": "102"
```

Comment:
Hmm. Someone skimming this could mistakenly think that MAIN is supported in 102. Could you add multiple ranges to clarify that 102 only introduced support for ISOLATED, and 128 also added support for MAIN?